examples/atari/iqn/train_iqn.py

```diff
@@ -21,6 +21,7 @@
 from chainerrl import misc
 from chainerrl import replay_buffer
 from chainerrl.wrappers import atari_wrappers
+import json
```

Comment:
In ChainerRL, import sentences are ordered as
- `__future__`
- `future` and `builtins`
- standard library in alphabetical order
- third-party (gym, chainer, numpy etc) in alphabetical order
- chainerrl in alphabetical order

So, can you move `json` to the standard library section?